suites/pacific/rgw/tier-2_rgw_archive.yaml

```diff
@@ -0,0 +1,233 @@
+# This test is to verify the archive zone with single site
```

Comment:
Please add a doc string indicating which conf is used and brief overview

Reply:
@Ameenasuhani the doc string and description added